Workbooks/ThycoticWorkbook.json

```diff
@@ -0,0 +1,322 @@
+            "version": "KqlParameterItem/1.0",
+            "name": "Timespan",
+            "type": 4,
+            "isRequired": true,
+            "value": {
+              "durationMs": 43200000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Most usage operations for SecretServer"
+      },
+      "name": "text - 9"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "cellValue": "page",
+            "linkTarget": "parameter",
+            "linkLabel": "Overview",
+            "subTarget": "FileType != \"test event\"",
+            "style": "primary"
+          },
+          {
+            "cellValue": "page",
+            "linkTarget": "parameter",
+            "linkLabel": "Secret",
+            "subTarget": "FileType == \"Secret\"",
+            "style": "primary"
+          },
+          {
+            "cellValue": "page",
+            "linkTarget": "parameter",
+            "linkLabel": "User",
+            "subTarget": "FileType == \"User\"",
+            "style": "primary"
+          },
+          {
+            "cellValue": "page",
+            "linkTarget": "parameter",
+            "linkLabel": "Folder",
+            "subTarget": "FileType == \"Folder\"",
+            "style": "secondary"
+          }
+        ]
+      },
+      "name": "links - 3",
+      "styleSettings": {
+        "margin": "0px",
+        "padding": "0px"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\n| where LogSeverity == 2 \n| where {page:query}\n| where TimeGenerated {Timespan:query}\n| summarize countRecord = count(), lastDate = arg_max(TimeGenerated, *) by FileName\n| order by countRecord\n| take 10\n| project FileType, Activity, SecretName=FileName, countRecord, lastDate ",
```

Comment:
Please add PROVIDER NAME and APPLIANCE NAME in the queries makes them return appropriate results for the connector.

Reply:
Same comment as the sample query, please make sure the query only targets logs from Thycotic

Reply:
Add params to query